collectors/UcsmChassisFaultCollector.py

```diff
@@ -13,13 +17,14 @@ def get_metrics(self):
         }
 
     def collect_metrics(self, server, handle):
+        logger.info("Collecting Metrics ")
         g = self.get_metrics()['faults']
 
         sys = handle.query_dn("sys")
         chassis = self.query(handle.query_children, sys, class_id="EquipmentChassis")
         for chassi in chassis:
             faults = self.query(handle.query_children, chassi, class_id="FaultInst")
-            print("Faults ..... ", faults, server)
+            logger.debug("Faults : %s %s", faults, server)
```

Comment:
personal taste, but I'd like to use an f-string here.

Reply:
Yes Its nice. I added that for most of them. please check.